clang/include/clang/Basic/AttrDocs.td

```diff
@@ -4254,6 +4254,10 @@ The [[intel::reqd_sub_group_size(n)]] attribute indicates that the kernel must
 be compiled and executed with a sub-group of size n. The value of n must be set
 to a sub-group size supported by the device, or device compilation will fail.
 
+The ``[[intel::sub_group_size(n)]]`` attribute has the same affect as the other
+forms of this, except it follows the SYCL 2020 Attribute Rules. See the
+``[[intel::named_sub_group_size(NAME)]]`` documentation for clarification.
+
```

Comment:
You should also mention that it is mutually exclusive with `named_sub_group_size`.